src/controllers/employee.js

```
@@ -1,180 +1,606 @@
+      "Oct",
+      "Nov",
+      "Dec",
+    ];
+    const data = [];
+    for (let i = 0; i < monthCount; i++) {
+      const d = { id: i + 1, month: months[i], value: 0 };
+      data.push(d);
+    }
+
+    //Iterate through the result
+    for (let result of results2) {
```

Biome

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)